### PR TITLE
Attempt to update openldap to v17 with tkldev

### PR DESCRIPTION
**Pull request by Christie018**
Used tkldev to create functional openldap.iso image with not error codes. Updated some config files to be compatible with refactors done to code in common directory repository. Added beginning of documentation directory, updated changelog with  all app version changes. Image tested with opensource GUI LDAP browser.

### CODE REVIEW

**Review by OnGle — APPROVED** [review]
Awesome, lgtm! :+1:

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 build
+docs
```
Comment (JedMeister) [inline .gitignore:2]: See my comments elsewhere about this.

**Review by JedMeister — COMMENTED** [review]
Awesome work Mattie. This is excellent. Just see my comment on the gitignore file - actually see the comment here: https://github.com/turnkeylinux-apps/etherpad/pull/16#discussion_r961137464

File: .gitignore (hunk 1 above)
Comment (Christie018) [reply .gitignore]: Went through and changed this for you.

**Pull request merged**